tools/benchmark/src/mocha/customOutputRunner.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Test } from "mocha";
+
+import type { BenchmarkDescription, MochaExclusiveOptions } from "../Configuration";
+import { timer } from "../timer";
+
+/**
+ * Options to configure a benchmark that reports custom measurements.
+ *
+ * @internal
+ */
+export interface CustomBenchmarkOptions extends MochaExclusiveOptions {
+	/**
+	 * Title about benchmark option
+	 */
+	title: string;
+
+	/**
+	 * Set of options to describe a benchmark.
+	 */
+	decription: BenchmarkDescription;
```

Comment:
If you want to aggregate all the various configuration into a single interface like CustomBenchmarkOptions, I suggest you do it like how the existing APIs in this package do. Rather than making callers do {title: "x", description: { type: ... } ...}}, you should extend the BenchmarkDescription interface like BenchmarkOptions does.